docs/source/api/apollo-server.md

```diff
@@ -315,13 +317,13 @@ This deprecated option is how you configured connecting Apollo Server to [Apollo
 
 ###### `plugins`
 
-`Array<PluginDefinition>`
+`Array`
 </td>
 <td>
 
-An array of plugins to install in your server. Each plugin definition is either an object with methods like `requestDidStart` and `serverWillStart`, or zero-argument function returning an object of that form. See [the full plugin documentation](../integrations/plugins/) for more details.
+An array of objects representing [plugins](../integrations/plugins) to install in your server instance.
```

Comment:
I think it's nice to be clear that the elements can either be plugin objects themselves or objects returning plugin objects (and the other page doesn't say that either).

(Not actually super clear why that's beneficial these days tbh... But it's true.)